src/common/util.cc

```diff
@@ -172,8 +195,37 @@ int get_cgroup_memory_limit(uint64_t *limit)
 out:
   fclose(f);
   return ret;
+  #else
+  return 0;
+  #endif
 }
 
+#ifdef _WIN32
+int get_windows_version(POSVERSIONINFOEXW ver) {
+    HMODULE hLib;
+    int ret_val = 0;
```

Comment:
could you move the definition of variables closer to where they are used for the first time for better readability?